src/com/google/enterprise/adaptor/SensitiveValueCodec.java

```diff
@@ -268,11 +271,27 @@ String getPrefix() {
    * -Dserver.keyAlias=adaptor \
    * -Dserver.secure=true
    * </pre>
+   *
+   * Example command line to run (passing the sensitive value on the command
+   * line):
+   * <pre>
+   * java \
+   * -Djavax.net.ssl.keyStore=keys.jks \
+   * -Djavax.net.ssl.keyStoreType=jks \
+   * -Djavax.net.ssl.keyStorePassword=changeit \
+   * -classpath 'adaptor-20130612-withlib.jar' \
+   * com.google.enterprise.adaptor.SensitiveValueCodec \
+   * -DsecurityLevel=ENCRYPTED \
+   * -Dserver.keyAlias=adaptor \
+   * -Dserver.secure=true
+   * 'sensitive value'
+   * </pre>
    */
   public static void main(String[] args) throws IOException {
     Config config = new Config();
     config.addKey("securityLevel", SecurityLevel.PLAIN_TEXT.toString());
     config.setValue("gsa.hostname", "not-used");
+    config.setValue("server.hostname", "not-used");
```

Comment:
The Google CLA should be under a corporate acceptance of TWT Business Solutions.